Siarhei_Surdo/02.Git.Local/log_online

```diff
@@ -0,0 +1,34 @@
+pi@u18sdevops1:~/it-study/02.Git.Local$ git checkout master
```

Comment:
missing git markup

Reply:
Add git markup. 
Change file extension for log_online 
Github don't see my git markup for file without extension.
Check please.